.github/workflows/ci.yml

```diff
@@ -119,5 +119,5 @@ jobs:
       - name: Test
         run: |
           tensorflow/lite/micro/tools/ci_build/test_makefile.sh
-          tensorflow/lite/micro/tools/ci_build/test_x86.sh
+          tensorflow/lite/micro/tools/ci_build/test_x86.sh tflite-micro/
```

Comment:
let's run this script from the parent directory of tflite-micro

Reply:
Done